utils/consts.go

```diff
@@ -2536,6 +2537,7 @@ const (
 	OptsSessionTTLLastUsage = "*sessionTTLLastUsage"
 	OptsSessionTTLUsage     = "*sessionTTLUsage"
 	OptsDebitInterval       = "*sessionDebitInterval"
+	OptsChargeable          = "*sessionChargeable"
```

Comment:
Please rename the prefix from *session to *sessions since it has to cover the module. Thanks!